stan/inc2prev.stan

```diff
@@ -155,11 +161,11 @@ model {
   // Priors for antibody model
   if (n_ab > 0) {
     init_dab ~ normal(init_ab_mean, init_ab_sd);
-    logit(beta) ~ normal(pbeta[1], pbeta[2]);
-    logit(gamma) ~ normal(pgamma_mean, pgamma_sd); 
-    logit(delta) ~ normal(pdelta[1], pdelta[2]);
-    log(k) ~ normal(0, 0.1);
-    log(l) ~ normal(0, 0.1);
+    logit_beta ~ normal(pbeta[1], pbeta[2]);
```

Comment:
I do think this avoids the issue for the logit space paramters but I am not very clear on why. Why is the LHS of them model block handled differently from the RHS of the TP block. Have you found anything on this that was helpful for getting this?

Reply:
I've found these forum posts helpful for understanding the issue and solutions a bit better:
https://discourse.mc-stan.org/t/do-i-need-jacobian-adjustment-here/19875/1
https://discourse.mc-stan.org/t/jacobian-in-reparameterizations/21592
https://discourse.mc-stan.org/t/basic-newbie-question-about-transforming-a-parameter-and-jacobians/1314
https://discourse.mc-stan.org/t/how-do-i-specify-a-logit-normal-distribution/4894/4